src/commands/PublishCommand.js

```diff
@@ -86,9 +86,11 @@ export const builder = {
     group: "Command Options:",
     describe: "Stop before actually publishing change to npm."
   },
-  "skip-temp-tag": {
+  "temp-tag": {
```

Comment:
`--use-temp-tag` scans a little better, here. Since it's a boolean option, you don't need to provide an explicit `default: false` since that's what happens anyway when the option is omitted.